generators/app/index.js

```diff
@@ -81,14 +81,13 @@ module.exports = yeoman.generators.Base.extend({
       author: this.props.authorName + ' <' + this.props.authorEmail + '>',
       main: 'lib/index.js',
       devDependencies: {
-        babel: '^5.6.0',
+        'babel-core': '^6.0.0',
+        'babel-preset-es2015': '^6.3.13',
+        'babel-preset-stage-0': '^6.3.13',
         mocha: '^2.2.5'
       },
       scripts: {
-        build: 'babel-plugin build',
-        push: 'babel-plugin publish',
-        test: 'babel-plugin test',
-        'test-mocha': 'mocha --compilers js:babel/register'
+        'test': 'mocha --compilers js:babel-core/register'
```

Comment:
Can you replace this with `babel-register`? `babel-core/register` will eventually be deprecated